examples/petsc/area.c

```diff
@@ -129,7 +129,7 @@ int main(int argc, char **argv) {
 
   // Setup DM
   if (read_mesh) {
-    ierr = DMPlexCreateFromFile(PETSC_COMM_WORLD, filename, PETSC_TRUE, &dm);
+    ierr = DMPlexCreateFromFile(PETSC_COMM_WORLD, filename, "mesh", PETSC_TRUE, &dm);
```

Comment:
passing `NULL` may be preferable so that `-dm_plex_distribute` works, versus `-mesh_dm_plex_distribute`. Though I think we'd be better off getting rid of this and use pure `DMCreate/DMSetType/DMSetFromOptions`.

Reply:
Agreed. I want to move to `DMCreate` eventually, I just wasn't thinking about that part here.